dart/lib/src/protocol/sentry_event.dart

```diff
@@ -31,10 +32,10 @@ class SentryEvent {
     this.userContext,
     this.contexts,
     this.breadcrumbs,
-    this.sdk,
   })  : eventId = eventId ?? SentryId.newId(),
-        platform = platform ?? sdkPlatform,
-        timestamp = timestamp ?? getUtcDateTime();
+        platform = platform ?? (isWeb ? browserPlatform : sdkPlatform),
```

Comment:
if this is going to be set here now, why do we need this on the client/transport? I believe this  should be done on the Client, like Java does

Reply:
✅ I moved it to the client.